server/routes/incentiveLevels.test.ts

```diff
@@ -860,8 +860,33 @@ describe('Incentive level management', () => {
           expect(incentivesApi.createIncentiveLevel).toHaveBeenCalled()
         })
     })
+    it('should sanitise incentive level name', () => {
+      // Given incentive level name should have no leading or preceding whitespaces
+      const expected: IncentiveLevel = {
+        code: 'EN5',
+        name: 'Enhanced 5',
+        active: true,
+        required: false,
+      }
+      incentivesApi.createIncentiveLevel.mockResolvedValue(expected)
+      const form: IncentiveLevelCreateData = {
+        formId: 'incentiveLevelCreateForm',
+        name: ' Enhanced 5   ',
+        code: 'EN5',
+      }
+      // When a user submits a form with a name that has contains whitespaces
+      // Then expect the whitespaces to have been removed
+      return request(app)
+        .post('/incentive-levels/add')
```

Comment:
This tests `POST:/incentive-levels/add`, you could also write a second test for the other endpoint updated in `server/routes/incentiveLevels.ts` which I think is `POST:/incentive-levels/edit/:levelCode`